aospy/calc.py

```diff
@@ -215,6 +215,30 @@ def __init__(self, calc_interface):
 
         self.data_out = {}
 
+    def _get_input_data_paths_one_dir_freq(self, name, data_in_direc, n=0):
+        """Get the names of netCDF files when all in same directory.
+        Allow the use to specify the separate output frequencies.
+        """
+        if isinstance(self.data_in_files[n][self.intvl_in][name], str):
+            data_in_files = [self.data_in_files[n][self.intvl_in][name]]
+        else:
+            data_in_files = self.data_in_files[n][self.intvl_in][name]
+        # data_in_files may hold absolute or relative paths
+        paths = []
+        for nc in data_in_files:
+            full = '/'.join([data_in_direc, nc]).replace('//', '/')
```

Comment:
The best way to do this is `os.path.join(data_in_direc, nc)` (only recently learned that...there's probably still aospy code that I wrote that still use the `'/'.join`!)